src/views/virtualmachines/details/tabs/configuration/storage/components/hooks/usePVCBound.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  modelToGroupVersionKind,
+  PersistentVolumeClaimModel,
+} from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+const usePVCBound = (pvcName: string, pvcNamespace: string): boolean => {
```

Comment:
there is already a call in const useDisksTableData for all pvcs, just expand it to return also the pvcs and in DiskList add to rowData a function to get the PVC by name and namespace , no need for this hook, its just add a call for each row

Reply:
So you are suggesting to use that call for getting all PVCs and just filter them by name, to get the specific PVC we need. Right? Then I'll need some helper function to check the PVC's status anyway.

Reply:
Yes u can create a function(kind of selector, no need for hook) or if u prefer, when u return the pvc's you can modify it to an object , in the matter of {[vmname: string]: pvcs[]} , then just pass this value of mapper to rowData as pvcProp={pvcObject[vmName]} , whatever u prefer, there are several example in our code for it, you can look at virtualmachinelist if you like

Reply:
Yes, thanks very much for the suggestion!